Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -6730,6 +6732,14 @@ - (BOOL)_shouldShowKeyboardForElementIgnoringInputMode:(const WebKit::FocusedEle
 
 - (void)_elementDidFocus:(const WebKit::FocusedElementInformation&)information userIsInteracting:(BOOL)userIsInteracting blurPreviousNode:(BOOL)blurPreviousNode activityStateChanges:(OptionSet<WebCore::ActivityState::Flag>)activityStateChanges userObject:(NSObject <NSSecureCoding> *)userObject
 {
+    if (_isRequestingAutocorrectionContext) {
+        RunLoop::main().dispatch([weakSelf = WeakObjCPtr<WKContentView> { self }, information, userIsInteracting, blurPreviousNode, activityStateChanges, userObject = RetainPtr { userObject } ] {
+            if (auto strongSelf = weakSelf.get())
+                [strongSelf _elementDidFocus:information userIsInteracting:userIsInteracting blurPreviousNode:blurPreviousNode activityStateChanges:activityStateChanges userObject:userObject.get()];
```

Comment:
what is the guarantee that `information` is still alive by this point?

Reply:
`information` is a copy here, rather than a reference (i.e. `&information`), so this should be safe.